seacrowd/sea_datasets/flores200/flores200.py

```diff
@@ -0,0 +1,498 @@
+    "pag_Latn",
+    "pan_Guru",
+    "pap_Latn",
+    "pes_Arab",
+    "pol_Latn",
+    "por_Latn",
+    "prs_Arab",
+    "pbt_Arab",
+    "quy_Latn",
+    "ron_Latn",
+    "run_Latn",
+    "rus_Cyrl",
+    "sag_Latn",
+    "san_Deva",
+    "sat_Olck",
+    "scn_Latn",
+    "shn_Mymr",
+    "sin_Sinh",
+    "slk_Latn",
+    "slv_Latn",
+    "smo_Latn",
+    "sna_Latn",
+    "snd_Arab",
+    "som_Latn",
+    "sot_Latn",
+    "spa_Latn",
+    "srd_Latn",
+    "srp_Cyrl",
+    "ssw_Latn",
+    "sun_Latn",
+    "swe_Latn",
+    "swh_Latn",
+    "szl_Latn",
+    "tam_Taml",
+    "tat_Cyrl",
+    "tel_Telu",
+    "tgk_Cyrl",
+    "tha_Thai",
+    "tir_Ethi",
+    "taq_Latn",
+    "taq_Tfng",
+    "tpi_Latn",
+    "tsn_Latn",
+    "tso_Latn",
+    "tuk_Latn",
+    "tum_Latn",
+    "tur_Latn",
+    "twi_Latn",
+    "uig_Arab",
+    "ukr_Cyrl",
+    "umb_Latn",
+    "urd_Arab",
+    "uzn_Latn",
+    "vec_Latn",
+    "vie_Latn",
+    "war_Latn",
+    "wol_Latn",
+    "xho_Latn",
+    "ydd_Hebr",
+    "yor_Latn",
+    "yue_Hant",
+    "zgh_Tfng",
+    "zsm_Latn",
+    "zul_Latn",
+]
+
+_SCRIPT_LANGUAGE_NAMES = ["Latn"]
+_LANGUAGE_NAMES = [lang for lang in _LANGUAGE_NAMES if (lang.split("_")[0] in _LANGUAGES) and (lang.split("_")[1] in _SCRIPT_LANGUAGE_NAMES)]
+
+_LICENSE = Licenses.CC_BY_NC_4_0.value
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: "https://github.com/openlanguagedata/flores/releases/download/v2.0-alpha.2/floresp-v2.0-alpha.2.zip",
+}
+
+_SPLITS = ["dev", "devtest"]
+
+_SENTENCES_PATHS = {lang: {split: os.path.join("floresp-v2.0-alpha.2", split, f"{split}.{lang}") for split in _SPLITS} for lang in _LANGUAGE_NAMES}
+
+_METADATA_PATHS = {split: os.path.join("floresp-v2.0-alpha.2", f"metadata_{split}.tsv") for split in _SPLITS}
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+_SUPPORTED_SCHEMA_STRINGS = [f"seacrowd_{str(TASK_TO_SCHEMA[task]).lower()}" for task in _SUPPORTED_TASKS]
+
+_SCHEMAS = [str(TASK_TO_SCHEMA[task]) for task in _SUPPORTED_TASKS]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+@dataclass
+class Flores200SeacrowdConfig(SEACrowdConfig):
+    """BuilderConfig for Nusantara."""
+
+    first_language_name: str = None
+    second_language_name: str = None
+    script_language: str = None
+
+
+class Flores200(datasets.GeneratorBasedBuilder):
+    """
+    The creation of FLORES-200 doubles the existing language coverage of FLORES-101.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = []
+
+    for script_names in _SCRIPT_LANGUAGE_NAMES:
+        for first_lang in _LANGUAGES:
+            first_lang_name = f"{first_lang}_{script_names}"
+
+            for second_lang in _LANGUAGES:
+
+                if first_lang == second_lang:
+                    continue
+
+                second_lang_name = f"{second_lang}_{script_names}"
+
+                subset_id = f"{first_lang}_{second_lang}_{script_names}"
```

Comment:
Hi @Enliven26, may I know why you didn't use `subset_id = f"{first_lang_name}_{second_lang_name}"`?